identity-iota/src/document/iota_document.rs

```diff
@@ -242,6 +242,11 @@ impl IotaDocument {
     Ok(())
   }
 
+  /// Returns whether the given method can be used to sign document updates.
+  pub fn is_signing_method(method: &IotaVerificationMethod) -> bool {
+    method.key_type() == MethodType::Ed25519VerificationKey2018
+  }
```

Comment:
This should probably use `UPDATE_METHOD_TYPES` or `IotaVerificationMethod::is_signing_method` if that's not removed. I was more thinking that this could be something like `resolve_signing_method` to perform the capability invocation method check too. If not, then `check_signing_method` should be updated to use `is_signing_method` internally to avoid duplicate code.

Reply:
Addressed the duplication. I changed it to only require the `MethodType`, because of problems converting the `method` in `check_signing_method` to an `IotaVerificationMethod` due to generics.